doc/conf.py

```diff
@@ -4,10 +4,10 @@
 """
 
 from documenteer.sphinxconfig.stackconf import build_package_configs
-import lsst.scriptrunner.version
+import scriptloader.version
```

Comment:
Good question. I've been avoiding it because T&S doesn't use any of DM, including the bit that allows shared sub-namespaces. But I think we should revisit that. I would also prefer to see DM code and T&S code in separate sub namespaces, since DM has so many packages.

Reply:
Also I think T&S mostly uses ts_ name prefix for this reason. I should start using that, and will use it for the queue package.